src/Selection.jl

```diff
@@ -2,6 +2,54 @@ module Selection
 
 using ..Types
 
-# TODO
+function rouletteWheelSelection(population::Vector{Vector{Int64}}, fitness_scores::Vector{Float64}, rand_generator::Function = rand)::Tuple{Vector{Int64}, Vector{Int64}}
```

Comment:
Why do we need a rand_generator? Just for testing?

Reply:
-> Just looked up how to test random stuff. Your way seems fine.

Reply:
yes its for testing, i didn,t find a propper way to inject a custom generator